### PR TITLE
Vertex AI Experiments - All-in-one notebook

### PR DESCRIPTION
**Pull request by inardini**
<br>
This PR is about an end-to-end notebook to show main capabilities of Vertex AI Experiments.
<br>
<br>
If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:

- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.

- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
<br>

### CODE REVIEW

File: notebooks/community/experiments/vertex_ai_model_experimentation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1750 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>Frameworks -&gt; frameworks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>Model Experimentation -&gt; lowercase</p><p> </p><p>Vertex Pipeline -&gt; Vertex AI Pipeline</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>This works on workbench, but not Colab. See template for version that works on both.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (andrewferlitsch) [inline notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>Add a bit of text what each helper function does</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (andrewferlitsch) [inline notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>combine 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (andrewferlitsch) [inline notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>heading sizes should not be smaller than the text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (andrewferlitsch) [inline notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>combine 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Comment (andrewferlitsch) [inline notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>maybe I am missing something. Shouldn't the code for tracking params in the experiment be here?</p><p> </p><p>What about creating Dataset artifacts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Comment (inardini) [reply notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>I use the last template. Could you point me out to that?</p>

Comment (inardini) [reply notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>fixed</p>

Comment (inardini) [reply notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: <p>Let me fix the dataset. About the parameters they are already tracked at that point. </p>

Comment (inardini) [reply notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: done

Comment (inardini) [reply notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: Also I tested that notebook on Colab. It works fine.

Comment (andrewferlitsch) [reply notebooks/community/experiments/vertex_ai_model_experimentation.ipynb:1]: Ahh, I see with the current template there is a sequence that works on both Workbench and Colab.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**